mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLOps.td

```diff
@@ -267,7 +276,8 @@ def SYCLMemref : AnyTypeOf<[
   NDItemMemRef,
   GroupMemRef,
   VecMemRef,
-  AtomicMemRef
+  AtomicMemRef,
```

Comment:
I'd prefer if we could have them in alphabetical order.

Reply:
Done